<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="Tab_Widget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Extraction</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <layout class="QGridLayout" name="Extraction_GridLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item row="0" column="2" rowspan="6" colspan="5">
             <layout class="QVBoxLayout" name="Matplot_VLayout"/>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QPushButton" name="process_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>process</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" rowspan="3" colspan="2">
             <layout class="QVBoxLayout" name="File_Explorer_VLayout">
              <item>
               <widget class="QTreeWidget" name="File_Explore_List_Widget">
                <column>
                 <property name="text">
                  <string notr="true">1</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="Category_ComboBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="Company_ComboBox"/>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLineEdit" name="search_company_name_lineBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Data System</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item row="2" column="2">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="6">
             <widget class="QTableWidget" name="tableWidget"/>
            </item>
            <item row="0" column="1" colspan="3">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QComboBox" name="comboBox_2"/>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QComboBox" name="comboBox_3"/>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_New_Company"/>
    <addaction name="actionOpen_Working_Space_Directory"/>
    <addaction name="actionWork_Space"/>
    <addaction name="separator"/>
    <addaction name="actionImport_PDF_JPG"/>
    <addaction name="Export_For_Current_Company"/>
    <addaction name="actionExport_All_Company_to_Excel"/>
    <addaction name="actionView_All_Image"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="actionImport_New_Company">
   <property name="text">
    <string>Open Company Folder</string>
   </property>
  </action>
  <action name="actionImport_PDF_JPG">
   <property name="text">
    <string>Import PDF/JPG</string>
   </property>
  </action>
  <action name="actionWork_Space">
   <property name="text">
    <string>Add Work Space Directory</string>
   </property>
  </action>
  <action name="view_image">
   <property name="text">
    <string>View All Image From Current Company</string>
   </property>
  </action>
  <action name="actionCheck_Images_With_Position_Data">
   <property name="text">
    <string>Check Images With Position Data</string>
   </property>
  </action>
  <action name="actionCurrent_Selected_Company">
   <property name="text">
    <string>Current Selected Company</string>
   </property>
  </action>
  <action name="actionAll_Company">
   <property name="text">
    <string>All Company</string>
   </property>
  </action>
  <action name="actionGuidence">
   <property name="text">
    <string>Guidence</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>?</string>
   </property>
  </action>
  <action name="Current_Selected_Company_Export">
   <property name="text">
    <string>Current Selected Company</string>
   </property>
  </action>
  <action name="actionAll_Company_2">
   <property name="text">
    <string>All Company</string>
   </property>
  </action>
  <action name="actionOpen_Working_Space_Directory">
   <property name="text">
    <string>Open Working Space Directory</string>
   </property>
  </action>
  <action name="Export_For_Current_Company">
   <property name="text">
    <string>Export Current Company to Excel</string>
   </property>
  </action>
  <action name="actionExport_All_Company_to_Excel">
   <property name="text">
    <string>Export All Company to Excel</string>
   </property>
  </action>
  <action name="actionView_All_Image">
   <property name="text">
    <string>View All Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
